backend/fms_core/template_paths.py

```diff
@@ -20,5 +20,5 @@
 CONTAINER_RENAME_TEMPLATE = static("submission_templates/Container_rename_v0.1.xlsx")
 SAMPLE_EXTRACTION_TEMPLATE = static("submission_templates/Extraction_v0.11.xlsx")
 SAMPLE_SUBMISSION_TEMPLATE = static("submission_templates/Sample_submission_v0.14.xlsx")
-SAMPLE_UPDATE_TEMPLATE = static("submission_templates/Sample_update_v0.5.xlsx")
+SAMPLE_UPDATE_TEMPLATE = static("submission_templates/Sample_update_v0.6.xlsx")
```

Comment:
If we are updating the templates, we should also update the csv/xlsx from our unit tests folder (fms_core/example_data) and from the front-end (cypress/fixtures) (and their links)

Reply:
yeah i'll get on it...